LLVM 3.1 compatibility issue

```
% make
cfg: build triple x86_64-apple-darwin
cfg: host triples x86_64-apple-darwin
cfg: target triples x86_64-apple-darwin
cfg: enabling more debugging (CFG_ENABLE_DEBUG)
cfg: host for x86_64-apple-darwin is x86_64
cfg: os for x86_64-apple-darwin is apple-darwin
cfg: good valgrind for x86_64-apple-darwin is 
cfg: using CC=clang (CFG_CC)
cfg: using CXX=clang++ (CFG_CXX)
cfg: enabling valgrind run-pass tests (CFG_ENABLE_VALGRIND_RPASS)
cfg: no lualatex found, deferring to xelatex
cfg: no xelatex found, deferring to pdflatex
cfg: no pdflatex found, disabling LaTeX docs
make: llvm
llvm[2]: Compiling CrashRecoveryContext.cpp for Release+Asserts build
In file included from /usr/local/Cellar/rust/nightly/src/llvm/lib/Support/CrashRecoveryContext.cpp:10:
/usr/local/Cellar/rust/nightly/src/llvm/include/llvm/Support/CrashRecoveryContext.h:169:3: error: only virtual member functions can be marked 'override'
  void recoverResources() override { delete this->resource; }
  ^                       ~~~~~~~~
/usr/local/Cellar/rust/nightly/src/llvm/include/llvm/Support/CrashRecoveryContext.h:182:3: error: only virtual member functions can be marked 'override'
  void recoverResources() override { this->resource->Release(); }
  ^                       ~~~~~~~~
2 errors generated.
make[2]: *** [/usr/local/Cellar/rust/nightly/x86_64-apple-darwin/llvm/lib/Support/Release+Asserts/CrashRecoveryContext.o] Error 1
make[1]: *** [all] Error 1
make: *** [/usr/local/Cellar/rust/nightly/x86_64-apple-darwin/llvm/Release+Asserts/bin/llvm-config] Error 2
```

--- a/configure
+++ b/configure
@@ -875,7 +875,7 @@ then
             | cut -d ' ' -f 2)
 
         case $CFG_CLANG_VERSION in
-            (3.0svn | 3.0 | 3.1* | 3.2* | 3.3* | 3.4* | 3.5* | 3.6*)
+            (3.2* | 3.3* | 3.4* | 3.5* | 3.6*)
             step_msg "found ok version of CLANG: $CFG_CLANG_VERSION"
             if [ -z "$CC" ]
             then

